types/mongodb/index.d.ts

```diff
@@ -2183,6 +2183,7 @@ export interface FindOneOptions<T> {
     snapshot?: boolean;
     timeout?: boolean;
     tailable?: boolean;
+    awaitData?: boolean;
```

Comment:
There's a related property `maxAwaitTimeMS`, which should be used with `awaitData`.